sections/semantics-sections.include

```diff
@@ -1255,10 +1255,10 @@
   The <code>h1</code>–<code>h6</code> elements are headings.
 
   The first element of <a>heading content</a> in an element of <a>sectioning content</a>
-  <a>represents</a> the heading for that (explicit) section. Subsequent headings of equal or higher
-  <a>rank</a> start new (implied) subsections that are part of the previous section's parent section.
-  Subsequent headings of lower <a>rank</a> start new (implied) subsections that are part of the previous
-  one. In both cases, the element <a>represents</a> the heading of the implied section.
+  <a>represents</a> the heading for that explicit section. Subsequent headings of equal or higher <a>rank</a>
+  start new implied subsections that are part of the previous section's parent section. Subsequent
+  headings of lower <a>rank</a> start new implied subsections that are part of the previous one.
```

Comment:
It took a little time to grasp it, but that last part made me smile. Thanks.

-- A little Off Topic:

Agreed, it doesn't make sense to start nit-picking, because, without wanting to step on anyone's toes, the whole chapter (4.3.9) could use a complete overhaul including a better structure: If interested see [my personal notes](https://github.com/rehierl/html-outliner-spec/blob/master/outliner-spec/outliner-notes.md) (I hope the licensing notice is acceptable?)